tests/test_content.py

```diff
@@ -352,3 +352,31 @@ def __format__(self, format):
     new_result = '{:open-the-pod-bay-doors}'.format(HAL9000())
 
     assert new_result == "I'm afraid I can't do that."  # output
+
+
+def test_side_note():
```

Comment:
I'm not completely sure I understand what you want to explain with this example. It it just the different options you have with

```
{0} {1}
```

vs 

```
{} {}
```

?